### ISSUE TITLE
Comparison between two YoloV3

### ISSUE THREAD

**Issue by hnhuang** [AlexeyAB/darknet#1004]
Hi Alexey,

I compared the your implementation of yoloV3 and the original one using the same weight file, cfg file. It seems that the original one is far better in some cases. For example:
![predictions_alexey](https://user-images.githubusercontent.com/4179004/41134290-38335e38-6afe-11e8-84ed-2b21b50f069b.jpg)
![predictions_origin](https://user-images.githubusercontent.com/4179004/41134291-387bb73c-6afe-11e8-9712-65cf86513eb7.jpg)

**Comment by hnhuang** [comment]
Your implementation only detects two character of an address. The original one can detect 13 characters.

**Comment by hnhuang** [comment]
I am using the original one to train the neural network. Is this the cause of different outputs?

**Comment by AlexeyAB** [comment]
@hnhuang Yes. Try to train network by using this repository and check mAP.
